File: JAVA/src/_Array/_ArrayQues/ReverseArray.java
package _Array._ArrayQues;

import java.util.Scanner;


public class ReverseArray {

    public static void Reverse(int size, int [] arr)
    {
        for(int i=0;i<=size/2;i++)
        {
         int temp = arr[i];
         arr[i]=arr[size-i-1];
         arr[size-i-1]=temp;
        }
    }

    public static void main(String[] args)
    {
        Scanner inp = new Scanner(System.in);
        int size = inp.nextInt();
        int arr[] = new int[size];
        for (int i=0;i<size;i++)
        {
            arr[i]=inp.nextInt();
        }
        Reverse(size,arr);
        for (int i=0;i<size;i++)
        {
            System.out.print(arr[i]+" ");
        }


    }
}

File: CPP/Array/reverseArray.cpp
#include <iostream>
using namespace std  ;
int reverse(int arr[],int size)
{
    for (int i=0;i<size/2;i++)
    {
        int temp = arr[i];
        arr[i]=arr[size-1-i];
        arr[size-1-i]=temp;
    }
}


void printArray(int arr[], int n) {
    
    for(int i=0; i<n; i++) {
        cout << arr[i] << " ";
    }
    cout << endl;
}
int main(){
    int size; 
    cin>>size;
    int arr[size];
    for (int i = 0;i<size;i++)
    {
      cin>>arr[i];
    }
   reverse(arr,size);
   
   printArray(arr,size);
}